### ISSUE TITLE
Proposal:  Using a PK/PD model for simulation-based assessment of probability of technical success in drug development.

### ISSUE THREAD

**Issue by kylebaron** [isop-phmx/studyGroup#22]
- General Topic: Using a PK/PD model for simulation-based assessment of probability of technical success in drug development.
- YouTube Live Feed Link: https://www.youtube.com/watch?v=F_oM1V4Lp90
- Potential Presenter(s): @kylebmetrum
- Potential Date(s): June 24, 2016
- Time: Noon EDT
- Background Resources for the Audience: [https://github.com/metrumresearchgroup/mrgsolve/](https://github.com/metrumresearchgroup/mrgsolve) [http://mrgsolve.github.io/user_guide/](http://mrgsolve.github.io/user_guide/)
- Vignette for the session [here](https://github.com/isop-phmx/studyGroup/files/331189/pts.pdf)
- `R` code, model, and data for the session [here](https://github.com/mrgsolve/examples/tree/master/PrTS)

**Comment by vjd** [comment]
hi Kyle! is this going to be a real world example where a published work can be  followed, or will you be providing some background material for folks to have a guess on the approach you may take. 

Great proposal! 👍

**Comment by kylebaron** [comment]
Do you want to do this June 10?

**Comment by vjd** [comment]
Can you do this the following week. Folks will be just returning from PAGE.
It doesn't have to be a Friday ..

**Comment by ghost** [comment]
Yeah ... I'll look for a day the next week.  Thanks for catching that.

KTB

On Thu, Jun 2, 2016, at 06:48, Vijay Ivaturi wrote:

> Can you do this the following week. Folks will be just returning
> from PAGE.
>  It doesn't have to be a Friday ..
> 
>  On Thu, Jun 2, 2016, 6:55 AM Kyle Baron
>  notifications@github.com wrote:
> 
> > Do you want to do this June 10?
> > 
> > —
> > You are receiving this because you commented.
> > Reply to this email directly, view it on GitHub
> > https://github.com/isop-phmx/studyGroup/issues/22#issuecomment-223195386
> > ,
> > or mute the thread
> > https://github.com/notifications/unsubscribe/ABXAmpbzxOHJBTcP6jBgNWIk_K6obsZEks5qHmJZgaJpZM4IWrZY
> > .
> > 
> >  --
> >  ~Vijay
> 
> — You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub[1], or mute the
> thread[2].

Links:
1. https://github.com/isop-phmx/studyGroup/issues/22#issuecomment-223268703
2. https://github.com/notifications/unsubscribe/AOo7HO3zxYiIn-g4R9dSOgUaabbAJX5Dks5qHsMLgaJpZM4IWrZY

**Comment by kylebaron** [comment]
@vjd June 24?

**Comment by vjd** [comment]
yup! lets lock it down

**Comment by vjd** [comment]
@kylebmetrum can you please setup a youtube link so that we can be ready to send out an email blast on Monday

**Comment by MarioGS** [comment]
None of the links: https://www.youtube.com/watch?v=TqEBhJ1kK3A 
or 
https://www.youtube.com/watch?v=ZGv1T9xp9uA&t=0s
worked for me. They last less than 5 minutes. Could you please upload the full session? Thanks :)

**Comment by kylebaron** [comment]
https://www.youtube.com/watch?v=F_oM1V4Lp90

**Comment by MarioGS** [comment]
Thanks Kyle :+1:

**Comment by kylebaron** [comment]
The `BMI` summaries for the two data sets should have been:

``` r
> data %>% group_by(set) %>% summarise(mean=mean(BMI))
Source: local data frame [2 x 2]

    set     mean
  <dbl>    <dbl>
1     1 26.95538
2     2 29.54463
```

**Comment by kylebaron** [comment]
I mentioned that the `PTS` literature I was using came from Pfizer pharamcometrics.  Ken Kowalski and Matt Hutmacher were already at [Ann Arbor Pharmacometrics Group](http://web.a2pg.com/)

**Comment by blalovic** [comment]
Hi Kayle, 

Really nice presentation on Friday.  Thank you!  Will you be able to share the dataset and the .R? 

Thanks, 
Bojan

**Comment by kylebaron** [comment]
Thanks, @blalovic .  I **believe** this will run if you get all the packages installed: https://github.com/mrgsolve/examples/tree/master/PrTS

People can clone the repo or just download a `.zip` file here: https://github.com/mrgsolve/examples

Let me know via issue tracker if something isn't there that is required to run and I'll get it updated.
